mindsdb/integrations/handlers/footballApi_handler/footballApi_handler.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import pandas as pd
+from mindsdb_sql import parse_sql
+
+from mindsdb.integrations.handlers.footballApi_handler.players_table import PlayersTable
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse, HandlerResponse,
+)
+from mindsdb.utilities import log
+from footballAPIClient import footballAPI, FootballAPI
+
+
+class FootballApiHandler(APIHandler):
+    """The Football API handler implementation"""
+
+    def __init__(self, name: str = None, **kwargs):
+        """Initialize the Football API handler.
+
+                Parameters
+                ----------
+                name : str
+                    name of a handler instance
+                """
+        super().__init__(name)
+        self.api_key = None
+        self.account_type = None
+
+        connection_data = kwargs.get('connection_data', {})
+        if 'api_key' in connection_data:
+            self.api_key = connection_data['api_key']
+        elif 'API_KEY' in os.environ:
```

Comment:
Since we have many integrations that use API keys, I think we need to be a little more specific on the name of the environment variable used. In order to extract the API key, can you please use `get_api_key()` method from `mindsdb.integrations.utilities.handler_utils`? This has been created for this purpose.